salt/metalk8s/defaults.yaml

```diff
@@ -37,6 +37,14 @@ kubeadm_preflight:
       - coreutils             # provides touch
 
 repo:
+  conflicting_packages:
+    # List of package that conflict with MetalK8s installation
+    # <package_name>: [<version1>, <version2>]
+    # Is version list is None then we consider all versions as conflicting
+    docker: null
+    docker-ce: null
+    containerd.io: null
+
```

Comment:
Could you take a look at the list provided by @baitanik in https://github.com/scality/metalk8s/discussions/2989#discussioncomment-224790 and check whether this list should be extended accordingly?

Reply:
Yes I could try with those packages, but for pip packages, it's a bit hard to check IMHO.
E.g.: Checking all dependencies of Salt installed by pip and not by system package... It's doable but not sure it's worth and honestly deserve another PR since it's really a different check to me